.github/workflows/terragrunt-deploy.yml

```diff
@@ -138,6 +138,13 @@ jobs:
           WFPREV_API_MEMORY: ${{vars.WFPREV_API_MEMORY}}
           WFPREV_API_PORT: ${{vars.WFPREV_API_PORT}}
           TARGET_AWS_ACCOUNT_ID: ${{secrets.TARGET_AWS_ACCOUNT_ID}}
+          WFPREV_CLIENT_ID: ${{vars.WFPREV_CLIENT_ID}}
+          WFPREV_CLIENT_SECRET: ${{secrets.WFPREV_CLIENT_SECRET}}
+          WEBADE_OAUTH2_CHECK_TOKEN_URL: ${{vars.WEBADE_OAUTH2_CHECK_TOKEN_URL}}
+          WEBADE_OAUTH2_CHECK_AUTHORIZE_URL: ${{vars.WEBADE_OAUTH2_CHECK_AUTHORIZE_URL}}
+          WFPREV_DATASOURCE_URL: ${{vars.WFPREV_DATASOURCE_URL}}
+          WFPREV_DATASOURCE_USERNAME: ${{vars.WFPREV_USERNAME}}
```

Comment:
I think there are some duplicate variable set up here and variables.tf. ( #DB section below it) 
it wont break but we might want to do some housekeeping afterward.